okta/resource_template_email.go

```diff
@@ -0,0 +1,129 @@
+package okta
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+var translationResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"language": &schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"subject": &schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"template": &schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+	},
+}
+
+func resourceTemplateEmail() *schema.Resource {
+	return &schema.Resource{
+		// No point in having an exist function since templates always exist
+		Create: resourceTemplateEmailCreate,
+		Read:   resourceTemplateEmailRead,
+		Update: resourceTemplateEmailUpdate,
+		Delete: resourceTemplateEmailDelete,
```

Comment:
If templates always exist, is delete necessary?

Reply:
Right good question, that comment is actually old. The default template always exists but I later found out the custom template is distinct. So yea, I can actually add that exists function back in.

Reply:
Good catch btw bb4bfd286528dc65cea05430ccf6235e34735eb6